app.py

```diff
@@ -140,6 +149,34 @@ def delete_user():
         return response
 
 
+# purposeful for matching club ID to other club attributes; debugging
+@app.route('/clubs', methods=['GET'])
```

Comment:
i dont' think you need this based on my comments in the javascript file this is used in